jdbc/src/main/spotbugs/spotbugs-exclude.xml

```diff
@@ -18,4 +18,12 @@
     <Class name="software.amazon.timestream.jdbc.TimestreamTablesResultSet"/>
     <Method name="populateCurrentRows"/>
   </Match>
+  <Match>
+    <!--The exclude is added as currently driver does not support the prepared statement.
+    This is not a new exclude, as it has already been added for populateCurrentRows for
+    TimestreamTablesResultSet in this file.-->
+    <Bug pattern="SQL_NONCONSTANT_STRING_PASSED_TO_EXECUTE"/>
+    <Class name="software.amazon.timestream.jdbc.TimestreamSchemasResultSet"/>
+    <Method name="populateCurrentRows" params="software.amazon.timestream.jdbc.TimestreamConnection,java.lang.String" returns="void" />
```

Comment:
Not sure why this exclusion is needed and wondering how it is related to Prepared statement?. 
Will marking schemaPattern parameter as "final" resolve this issue?